cmd/sandbox-ce/compose/compose.go

```diff
@@ -2,16 +2,135 @@ package compose
 
 import (
 	"context"
+	"fmt"
+	"io"
+	"net/http"
 	"os"
 	"os/exec"
+	"path/filepath"
+	"strings"
+
+	"github.com/mitchellh/go-homedir"
+	"github.com/pkg/errors"
 )
 
-func Run(ctx context.Context, arg ...string) error {
-	cmd := exec.CommandContext(ctx, "docker-compose", arg...)
+// composeContainerPath is the url docker-compose is downloaded from in case
+// that it's not already present in the system
+const composeContainerPath = "https://github.com/docker/compose/releases/download/1.24.0/run.sh"
+
+var envKeys = map[string]bool{
```

Comment:
Why is this a `map`? It looks like it could simply be a `[]string`

Reply:
Addressed [here](https://github.com/smacker/superset-compose/pull/4/commits/5f9fc1a92dc709c2dfea3969dbc7b85542c7de66).